File: C/crescente.c
#include <stdio.h>
#include <stdlib.h>

int main(){
	
	int a,b;
	
	printf("Digite dois numeros:\n");
	scanf("%i", &a);
	scanf("%i", &b);
	
	do{
		if(a>b) printf("DECRESCENTE!\n");
		else printf("CRESCENTE!\n");
		
		printf("Digite outros dois numeros:\n");
		scanf("%i", &a);
		scanf("%i", &b);
		
	}while(a!=b);
	
	
	return 0;
}

File: C#/Crescente.cs
using System;
using System.Globalization;

public class Crescente{
	static void Main(string[] args){
		
		int a,b;
		
		Console.WriteLine("Digite dois numeros: ");
		a = int.Parse(Console.ReadLine());
		b = int.Parse(Console.ReadLine());
		
		while(a!=b){
			if(a<b) Console.WriteLine("CRESCENTE!");
			else Console.WriteLine("DECRESCENTE!");
			
			Console.WriteLine("Digite mais dois numeros: ");
			a = int.Parse(Console.ReadLine());
			b = int.Parse(Console.ReadLine());
		}
		
		
	}
}